File: cSharp/day45.cs

using System;


Console.WriteLine("Input messag");
string message = Console.ReadLine();

string[] badWords = new string[]{
    "fuck",
    "dead",
    "suck",
    "asshole",
    "faggot"
};

foreach (string badWord in badWords)
{
    message = message.Replace(
        badWord,
        new string('*', badWord.Length)
    );
}

Console.WriteLine(
    "Processed message: " +
    message
);

File: go/day45.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func square(x float64) float64 {
	return x * x
}

func main() {
	fmt.Print("Input message: ")

	reader := bufio.NewReader(os.Stdin)
	input, _ := reader.ReadString('\n')
	var message = strings.TrimSpace(input);

	var badWords = []string{
		"fuck",
		"dead",
		"suck",
		"asshole",
		"faggot",
	}

	
	for _, badWord := range badWords {
		message = strings.Replace(
			message,
			badWord,
			strings.Repeat("*",len(badWord)),
			-1,
		);
	}

	
	fmt.Print(
		"Result: " + 
		message +
		"\n\n",
	);

}